internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -141,12 +146,21 @@ func (v *vmConsoleProxy) Reconcile(request *common.Request) ([]common.ReconcileR
 		return nil, err
 	}
 
+	var cleanupResults []common.CleanupResult
+	oldApiServiceCleanup, err := deleteOldApiService(request)
```

Comment:
How about the v1.token.kubevirt.io APIService, is it somwhere deleted?

Reply:
Yes, it's deleted in `Cleanup()` method, on line 234:
```
	objectsToDelete = append(objectsToDelete,
		v.clusterRoleBinding.DeepCopy(),
		v.roleBinding.DeepCopy(),
		v.apiService.DeepCopy())

	cleanupResults, err := common.DeleteAll(request, objectsToDelete...)
```